src/pages/RFOX/hooks/useCurrentEpochRewardsQuery.ts

```diff
@@ -36,34 +40,37 @@ export const useCurrentEpochRewardsQuery = ({
         UseQueryResult<bigint, Error>,
       ],
     ) => {
-      const combineResults = (
-        _results: (Epoch[] | bigint | CurrentEpochMetadata | undefined)[],
-      ) => {
+      const combineResults = (_results: (Epoch[] | bigint | undefined)[]) => {
         if (!stakingAssetAccountAddress) return 0n
 
         const results = _results as EpochRewardsResultTuple
 
         const [epochHistory, currentEpochRewardUnits, affiliateRevenue] = results
 
         if (!epochHistory || !currentEpochRewardUnits || !affiliateRevenue || !currentEpochMetadata)
-          return
+          return 0n
```

Comment:
@NeOMakinG is the change of return heuristics expected here?

Reply:
I just unified the return type. Explicit `0` value instead of undefined fallback values resulting in `0` in the end regardless. Wasn't too calculated outside of that